components/sidebar/sidebar_channel_button_or_link/sidebar_channel_button_or_link_icon.jsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {Constants} from 'utils/constants.jsx';
+
+import StatusIcon from '../../status_icon.jsx';
```

Comment:
Can you import this as `components/status_icon.jsx` instead of using the relative path?

Reply:
Done